src/lib/common/utils.test.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from 'vitest';
+import * as utils from './utils';
+
+const TRUTHY_INPUT_DATA = ['TRUE', String(true), String(1)];
```

Comment:
Good observation. The `TRUTHY_INPUT_DATA` should definitely include a case for "any non-falsy string" since any string passed to `parseBool` that isn't falsy should return true. For example:
```ts
parseBool("example non-falsy string"); // Output: true
```

Reply:
Done <https://github.com/EthanThatOneKid/acmcsuf.com/commit/7b88b6be5fdff6bf021224e69fc0834b2adb490e>.

Reply:
poggers